src/devices/filesystemdevice.cpp

```diff
@@ -77,3 +78,7 @@ FilesystemDevice::~FilesystemDevice() {
   watcher_thread_->exit();
   watcher_thread_->wait();
 }
+
+void FilesystemDevice::WatcherError(const QString& error) {
```

Comment:
Can these just be lambdas in the `connect()` call?

Reply:
> Can these just be lambdas in the `connect()` call?

Yep

Reply:
Actually, can be done without a lambda.